lib/rollbar/request_data_extractor.rb

```diff
@@ -201,8 +201,8 @@ def rollbar_route_params(env)
       end
     end
 
-    def rollbar_request_session(rack_req)
-      session = rack_req.session
+    def rollbar_request_session(env)
+      session = env.fetch('rack.session', {})
```

Comment:
We just want the session, but we don't want to modify the underlying `env`, so in theory this will fetch the same value that `rack_req.session` does if it is already set, otherwise it returns an empty hash if it is not set (which is the same as `rack_req.session`). The only difference is that `env['rack.session']` does not get set to `{}` if it was undefined.